terraform/projects/app-ecs-services/templates/auth-proxy.conf.tpl

```diff
@@ -18,7 +18,7 @@ server {
     return 200 "Static health check";
   }
 
-  resolver 8.8.8.8 valid=10s;
+  resolver 10.0.0.2 valid=10s;
```

Comment:
we can DRY this up by specifying `resolver` at the higher level - it's a valid `http` directive, so doesn't have to be inside a `server` block